src/Core/Style.js

```diff
@@ -566,6 +569,9 @@ class Style {
     drawingStylefromContext(context) {
         const style = {};
         if (this.fill.color || this.fill.pattern || context.globals.fill) {
+            if (typeof this.fill.pattern === 'string') {
```

Comment:
> Currently drawingStylefromContext() is called before the instanciation of a Style, thus we should do that check here to be able to use the mapPropertiesFromContext(). 

Ok, so it would be doable in the following PR where `drawingStylefromContext` will instantiate a `Style` ?

> It's also to prevent a breaking change as in my opinon fill.pattern should always be an object describing the source image to use and the modification (crop and or color change) we want (or not) to apply.

I agree, we could choose this solution (with a deprecation message), also in the following PR, WDYT ?

Reply:
reolving too, related to another comment, will be handled later